### PR TITLE
feat(ad): create base ad crud

### PR DESCRIPTION
**Pull request by MisterAzix**
# Ticket Notion
https://www.notion.so/ETQ-dev-JV-cr-er-la-collection-posts-1d4410cee43040cc8393ef28a386fa6d
https://www.notion.so/ETQ-users-JVP-cr-er-un-post-4763999832fe448b864b24c2750df888
https://www.notion.so/ETQ-users-JVP-modifier-un-post-f2da901725ee47a89178e3d9766a954a
https://www.notion.so/ETQ-users-JVP-supprimer-mon-post-55646b87e3ae4b6fbd5aad4fe9661588

# Screen (si besoin)
image

# Autre information
- Thumbnails addition gonna be add in next PR
- Fixtures gonna be add after thumbnails addition system

### CODE REVIEW

**Review by PatRider64 — APPROVED** [review]

File: src/composer.json [hunk 1]
```diff
@@ -26,6 +26,7 @@
         "symfony/runtime": "6.1.*",
         "symfony/security-bundle": "6.1.*",
         "symfony/twig-bundle": "6.1.*",
+        "symfony/validator": "6.1.*",
```
Comment (justinDev91) [inline src/composer.json:29]: Where validator is need for this ticket ? @MisterAzix

File: src/src/Form/AdType.php [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class AdType extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options): void
+    {
+        $builder
+            ->add('title')
+            ->add('description')
+            ->add('price')
```
Comment (justinDev91) [inline src/src/Form/AdType.php:18]: Missing $thumbnails_urls field  !

File: src/templates/ad/index.html.twig [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+{% block title %}Ad index{% endblock %}
+
+{% block body %}
+    <h1>Ad index</h1>
+
+    <table class="table">
+        <thead>
+            <tr>
+                <th>Id</th>
```
Comment (justinDev91) [inline src/templates/ad/index.html.twig:11]: Don't need to display  id !

File: src/templates/ad/index.html.twig [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+                <th>Title</th>
+                <th>Price</th>
+                <th>Thumbnails_urls</th>
+                <th>actions</th>
+            </tr>
+        </thead>
+        <tbody>
+        {% for ad in ads %}
+            <tr>
+                <td>{{ ad.id }}</td>
```
Comment (justinDev91) [inline src/templates/ad/index.html.twig:21]: smt

File: src/templates/ad/show.html.twig [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+{% block title %}Ad{% endblock %}
+
+{% block body %}
+    <h1>Ad</h1>
+
+    <table class="table">
+        <tbody>
+            <tr>
+                <th>Id</th>
```
Comment (justinDev91) [inline src/templates/ad/show.html.twig:11]: Don't display id

File: src/templates/ad/show.html.twig [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+                <td>{{ ad.id }}</td>
```
Comment (justinDev91) [inline src/templates/ad/show.html.twig:12]: smt

**Review by justinDev91 — CHANGES_REQUESTED** [review]

File: src/composer.json (hunk 1 above)
Comment (MisterAzix) [reply src/composer.json:29]: It is needed when u use `make:crud` command, then it used in the form builder to validate images

File: src/src/Form/AdType.php (hunk 2 above)
Comment (MisterAzix) [reply src/src/Form/AdType.php:18]: Come in next PR as I said ;)

image

File: src/templates/ad/index.html.twig (hunk 3 above)
Comment (MisterAzix) [reply src/templates/ad/index.html.twig:11]: Sure, that just the base file generated by the `make:crud` command, gonna disappeared in front PR I guess ? :)

File: src/src/Form/AdType.php (hunk 2 above)
Comment (justinDev91) [reply src/src/Form/AdType.php:18]: Ok

File: src/templates/ad/index.html.twig (hunk 3 above)
Comment (justinDev91) [reply src/templates/ad/index.html.twig:11]: Yeah bv

**Review by justinDev91 — APPROVED** [review]

**Review by Triips-TheCoder — APPROVED** [review]

**Pull request merged**